scripts/export.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+# Export sales val flags to a CSV for upload to iasworld. Outputs the CSV
+# to stdout.
+#
+# Example usage:
+#
+#   python3 scripts/export.py > export.csv
+#
+import logging
+import os
+import sys
+
+import pandas
+import pyathena
+import pyathena.pandas.util
+
+PIN_FIELD = "PARID"
+SALE_KEY_FIELD = "SALEKEY"
+IS_OUTLIER_FIELD = "USER26"
+OUTLIER_TYPE_FIELD = "USER27"
+RUN_ID_FIELD = "USER28"
+ANALYST_DETERMINATION_FIELD = "USER29"
+ANALYST_REVIEW_DATE_FIELD = "UDATE1"
+
+OUTLIER_TYPE_CODES = {
+    "Anomaly (high)": "1",
+    "Anomaly (low)": "2",
+    "Family sale (high)": "3",
+    "Family sale (low)": "4",
+    "High price (raw & sqft)": "5",
+    "High price (raw)": "6",
+    "High price (sqft)": "7",
+    "High price swing": "8",
+    "Home flip sale (high)": "9",
+    "Home flip sale (low)": "10",
+    "Low price (raw & sqft)": "11",
+    "Low price (raw)": "12",
+    "Low price (sqft)": "13",
+    "Low price swing": "14",
+    "Non-person sale (high)": "15",
+    "Non-person sale (low)": "16",
+    "PTAX-203 flag (High)": "17",
+    "PTAX-203 flag (Low)": "17",
+    "Not outlier": pandas.NA,
```

Comment:
They seem reasonable to me, but I'm not 100% sure of these two choices, and we need to confirm them with Mirella:

* High and low PTAX-203 flags should share the same code, since there is only one `PTAX-203 flag` code (17)
* Sales that have been flagged as _not_ being outliers should be uploaded to iasWorld with `USER26 = 'N'` (not an outlier) and `USER27 = ""` (no reason code)